### ISSUE TITLE
executable 'servo_node_main' not found 

### ISSUE THREAD

**Issue by ravijo** [ros-planning/moveit2#2383]
### Description

I am trying to execute `ur_moveit.launch.py` on ROS Iron but getting the following exception:

```
[ERROR] [launch]: Caught exception in launch (see debug for traceback): executable 'servo_node_main' not found on the libexec directory '/opt/ros/iron/lib/moveit_servo' 
```

### Your environment
* ROS Distro: Iron
* OS Version: Ubuntu 22.04.3 LTS
* ROS Source or Binary build: Binary
* MoveIt Source or Binary build: Error occurs in both (source and binary)
* Which RMW (Fast DDS or Cyclone DDS): `rmw_cyclonedds_cpp`

### Steps to reproduce
```
$ ros2 launch ur_moveit_config ur_moveit.launch.py use_fake_hardware:=true ur_type:=ur5e use_sim_time:=true
[INFO] [launch]: All log files can be found below /home/user/.ros/log/2023-09-24-17-53-28-365043-dell-21428
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): executable 'servo_node_main' not found on the libexec directory '/opt/ros/iron/lib/moveit_servo' 
```

**Comment by ravijo** [comment]
This error does not appear in ROS Humble + MoveIt Humble Source, i.e., `git clone https://github.com/ros-planning/moveit2_tutorials -b humble --depth 1`

**Comment by ibrahiminfinite** [comment]
Hi @ravijo, thanks for reporting the bug!
 
MoveIt Servo recently had a major update, which is the version on the `main` branch of MoveIt2.
The `servo_node_main`  was an executable name used in the old version and that is why you get the error on Iron.
These changes are not on Humble branch, so you there would not be any error there.


Also, is this error from a package outside of MoveIt2 ? or from MoveIt2 examples?

**Comment by ravijo** [comment]
@ibrahiminfinite 

Thanks for the confirmation.

> Also, is this error from a package outside of MoveIt2 ? or from MoveIt2 examples?

Ya. It is coming from `ros2 launch ur_moveit_config ur_moveit.launch.py` which is invoking `moveit_servo` internally.

BTW, I came accoss another issue while following `moveit2_tutorials`. May I kindly request you to look at https://github.com/ros-planning/moveit2_tutorials/issues/767

Thank you so much!

**Comment by sjahr** [comment]
@ravijo I am closing this issue since it seemed to be answered. Feel free to re-open new ones for more questions!

**Comment by ravijo** [comment]
@sjahr 

Thanks for the followup. 

Another workaround for this issue is to create a symbolic link as shown below:

```
$ cd /home/user/ros2_ws/install/moveit_servo/lib/moveit_servo
$ ln -s servo_node servo_node_main
```

**Comment by wei1224hf** [comment]
> Hi @ravijo, thanks for reporting the bug!
> 
> MoveIt Servo recently had a major update, which is the version on the `main` branch of MoveIt2. The `servo_node_main` was an executable name used in the old version and that is why you get the error on Iron. These changes are not on Humble branch, so you would not have any error there.
> 
> Also, is this error from a package outside of MoveIt2 ? or from MoveIt2 examples?

I get this error from Moveit2 example , 

os2 launch moveit2_tutorials jupyter_notebook_prototyping.launch.py start_servo:=true
[INFO] [launch]: All log files can be found below /home/gene/.ros/log/2024-01-31-15-15-42-218572-gene-virtual-machine-7582
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): executable 'servo_node_main' not found on the libexec directory '/opt/ros/iron/lib/moveit_servo'

**Comment by ravijo** [comment]
@wei1224hf Please try running the humble version of the repository. For more information, please feel free to read the comments above

**Comment by shrutic12** [comment]
Hello @ravijo I am facing the same error 

`[INFO] [launch]: All log files can be found below /home/user/.ros/log/2024-04-10-16-56-58-476502-user-OptiPlex-790-8682
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): executable 'servo_node' not found on the libexec directory '/home/user/ws_moveit2/install/moveit_servo/lib/moveit_servo' 
`

I have tried the workaround you suggest 

`$ cd /home/user/ros2_ws/install/moveit_servo/lib/moveit_servo
$ ln -s servo_node servo_node_main
`
and I am able to launch the robot and moveit without any issue. But the `ros2 service call /servo_node/start_servo std_srvs/srv/Trigger {}` service is not available. Can you help me out? 

Here is the log I see on launching : 
`[INFO] [launch]: All log files can be found below /home/user/.ros/log/2024-04-10-16-59-31-237483-user-OptiPlex-790-9396
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [ur_ros2_control_node-1]: process started with pid [9413]
[INFO] [robot_state_publisher-2]: process started with pid [9415]
[INFO] [spawner-3]: process started with pid [9417]
[INFO] [move_group-4]: process started with pid [9419]
[INFO] [servo_node-5]: process started with pid [9422]
[INFO] [spawner-6]: process started with pid [9436]
[INFO] [spawner-7]: process started with pid [9439]
[INFO] [spawner-8]: process started with pid [9441]
[INFO] [spawner-9]: process started with pid [9443]
[INFO] [spawner-10]: process started with pid [9445]
[INFO] [spawner-11]: process started with pid [9447]
[INFO] [spawner-12]: process started with pid [9449]
[INFO] [spawner-13]: process started with pid [9457]
[ur_ros2_control_node-1] [WARN] [1712761173.354705101] [controller_manager]: [Deprecated] Passing the robot description parameter directly to the control_manager node is deprecated. Use '~/robot_description' topic from 'robot_state_publisher' instead.
[ur_ros2_control_node-1] [INFO] [1712761173.361099233] [resource_manager]: Loading hardware 'ur10e' 
[ur_ros2_control_node-1] [INFO] [1712761173.370810550] [resource_manager]: Initialize hardware 'ur10e' 
[ur_ros2_control_node-1] [WARN] [1712761173.371063959] [mock_generic_system]: Parameter 'fake_sensor_commands' has been deprecated from usage. Use'mock_sensor_commands' instead.
[ur_ros2_control_node-1] [WARN] [1712761173.371237703] [mock_generic_system]: Parsing of optional initial interface values failed or uses a deprecated format. Add initial values for every state interface in the ros2_control.xacro. For example: 
[ur_ros2_control_node-1] <state_interface name="velocity"> 
[ur_ros2_control_node-1]   <param name="initial_value">0.0</param> 
[ur_ros2_control_node-1] </state_interface>
[ur_ros2_control_node-1] [INFO] [1712761173.372803987] [resource_manager]: Successful initialization of hardware 'ur10e'
[robot_state_publisher-2] [INFO] [1712761173.262008963] [robot_state_publisher]: got segment base
[robot_state_publisher-2] [INFO] [1712761173.262156432] [robot_state_publisher]: got segment base_link
[robot_state_publisher-2] [INFO] [1712761173.262170954] [robot_state_publisher]: got segment base_link_inertia
[robot_state_publisher-2] [INFO] [1712761173.262179600] [robot_state_publisher]: got segment camera_bottom_screw_frame
[robot_state_publisher-2] [INFO] [1712761173.262187590] [robot_state_publisher]: got segment camera_link
[robot_state_publisher-2] [INFO] [1712761173.262195147] [robot_state_publisher]: got segment flange
[robot_state_publisher-2] [INFO] [1712761173.262202677] [robot_state_publisher]: got segment forearm_link
[robot_state_publisher-2] [INFO] [1712761173.262210139] [robot_state_publisher]: got segment ft_frame
[robot_state_publisher-2] [INFO] [1712761173.262217535] [robot_state_publisher]: got segment shoulder_link
[robot_state_publisher-2] [INFO] [1712761173.262225036] [robot_state_publisher]: got segment tcp_screwdriver
[robot_state_publisher-2] [INFO] [1712761173.262232520] [robot_state_publisher]: got segment tool0
[robot_state_publisher-2] [INFO] [1712761173.262239815] [robot_state_publisher]: got segment upper_arm_link
[robot_state_publisher-2] [INFO] [1712761173.262247135] [robot_state_publisher]: got segment world
[robot_state_publisher-2] [INFO] [1712761173.262254460] [robot_state_publisher]: got segment wrist_1_link
[robot_state_publisher-2] [INFO] [1712761173.262261744] [robot_state_publisher]: got segment wrist_2_link
[robot_state_publisher-2] [INFO] [1712761173.262268907] [robot_state_publisher]: got segment wrist_3_link
[ur_ros2_control_node-1] [INFO] [1712761173.377254514] [resource_manager]: 'configure' hardware 'ur10e' 
[ur_ros2_control_node-1] [INFO] [1712761173.377293830] [resource_manager]: Successful 'configure' of hardware 'ur10e'
[ur_ros2_control_node-1] [INFO] [1712761173.378522076] [resource_manager]: 'activate' hardware 'ur10e' 
[ur_ros2_control_node-1] [INFO] [1712761173.378554197] [resource_manager]: Successful 'activate' of hardware 'ur10e'
[servo_node-5] [WARN] [1712761173.400255188] [moveit_servo.servo_node]: Intra-process communication is disabled, consider enabling it by adding: 
[servo_node-5] extra_arguments=[{'use_intra_process_comms' : True}]
[servo_node-5] to the Servo composable node in the launch file
[servo_node-5] [INFO] [1712761173.435964228] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.019787 seconds
[servo_node-5] [INFO] [1712761173.436311093] [moveit_robot_model.robot_model]: Loading robot model 'ur'...
[servo_node-5] [INFO] [1712761173.436331212] [moveit_robot_model.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[move_group-4] [INFO] [1712761173.532976373] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.0258001 seconds
[move_group-4] [INFO] [1712761173.533080962] [moveit_robot_model.robot_model]: Loading robot model 'ur'...
[move_group-4] [INFO] [1712761173.533097927] [moveit_robot_model.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[servo_node-5] [INFO] [1712761173.874458907] [moveit_ros.current_state_monitor]: Listening to joint states on topic '/joint_states'
[servo_node-5] [INFO] [1712761173.926433544] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/attached_collision_object' for attached collision objects
[servo_node-5] [INFO] [1712761173.926466620] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting planning scene monitor
[servo_node-5] [INFO] [1712761173.933764244] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/planning_scene'
[servo_node-5] [INFO] [1712761173.935465961] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Publishing maintained planning scene on '/servo_node/publish_planning_scene'
[move_group-4] [INFO] [1712761173.941923803] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[move_group-4] [INFO] [1712761173.942114189] [moveit.ros_planning_interface.moveit_cpp]: Listening to 'joint_states' for joint states
[move_group-4] [INFO] [1712761173.942936182] [moveit_ros.current_state_monitor]: Listening to joint states on topic 'joint_states'
[move_group-4] [INFO] [1712761173.951587984] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/attached_collision_object' for attached collision objects
[move_group-4] [INFO] [1712761173.951621138] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting planning scene monitor
[move_group-4] [INFO] [1712761173.955765628] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/planning_scene'
[move_group-4] [INFO] [1712761173.955795718] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[move_group-4] [INFO] [1712761173.956409112] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to 'collision_object'
[move_group-4] [INFO] [1712761173.956877075] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to 'planning_scene_world' for planning scene world geometry
[servo_node-5] [ERROR] [1712761173.960767637] [moveit_robot_model.robot_model]: Group 'ur_manipulator' not found in model 'ur'
[servo_node-5] [ERROR] [1712761173.960801485] [moveit_servo.servo_calcs]: Invalid move group name: `ur_manipulator`
[servo_node-5] terminate called after throwing an instance of 'std::runtime_error'
[servo_node-5]   what():  Invalid move group name
[servo_node-5] Stack trace (most recent call last):
[servo_node-5] #14   Object "", at 0xffffffffffffffff, in 
[servo_node-5] #13   Object "/home/user/ws_moveit2/install/moveit_servo/lib/moveit_servo/servo_node_main", at 0x5568c2744894, in _start
[move_group-4] [WARN] [1712761173.983691707] [moveit.ros.occupancy_map_monitor.middleware_handle]: Resolution not specified for Octomap. Assuming resolution = 0.1 instead
[move_group-4] [ERROR] [1712761173.983726311] [moveit.ros.occupancy_map_monitor.middleware_handle]: No 3D sensor plugin(s) defined for octomap updates
[move_group-4] [INFO] [1712761174.019212556] [moveit.ros_planning_interface.moveit_cpp]: Loading planning pipeline 'pilz_industrial_motion_planner'
[move_group-4] [INFO] [1712761174.069646183] [moveit.pilz_industrial_motion_planner.joint_limits_aggregator]: Reading limits from namespace robot_description_planning
[move_group-4] [INFO] [1712761174.119914932] [moveit.pilz_industrial_motion_planner]: Available plugins: pilz_industrial_motion_planner/PlanningContextLoaderCIRC pilz_industrial_motion_planner/PlanningContextLoaderLIN pilz_industrial_motion_planner/PlanningContextLoaderPTP 
[move_group-4] [INFO] [1712761174.119944610] [moveit.pilz_industrial_motion_planner]: About to load: pilz_industrial_motion_planner/PlanningContextLoaderCIRC
[move_group-4] [INFO] [1712761174.133041178] [moveit.pilz_industrial_motion_planner]: Registered Algorithm [CIRC]
[move_group-4] [INFO] [1712761174.133074810] [moveit.pilz_industrial_motion_planner]: About to load: pilz_industrial_motion_planner/PlanningContextLoaderLIN
[move_group-4] [INFO] [1712761174.145644195] [moveit.pilz_industrial_motion_planner]: Registered Algorithm [LIN]
[move_group-4] [INFO] [1712761174.145678570] [moveit.pilz_industrial_motion_planner]: About to load: pilz_industrial_motion_planner/PlanningContextLoaderPTP
[move_group-4] [INFO] [1712761174.150387135] [moveit.pilz_industrial_motion_planner]: Registered Algorithm [PTP]
[move_group-4] [INFO] [1712761174.150448130] [moveit.ros_planning.planning_pipeline]: Using planning interface 'Pilz Industrial Motion Planner'
[move_group-4] [INFO] [1712761174.177556571] [moveit.ros_planning_interface.moveit_cpp]: Loading planning pipeline 'chomp'
[move_group-4] [INFO] [1712761174.238325205] [moveit.ros_planning.planning_pipeline]: Using planning interface 'CHOMP'
[servo_node-5] #12   Source "../csu/libc-start.c", line 392, in __libc_start_main_impl [0x7fc5e736ce3f]
[servo_node-5] #11   Source "../sysdeps/nptl/libc_start_call_main.h", line 58, in __libc_start_call_main [0x7fc5e736cd8f]
[servo_node-5] #10   Object "/home/user/ws_moveit2/install/moveit_servo/lib/moveit_servo/servo_node_main", at 0x5568c2744753, in main
[servo_node-5] #9    Object "/home/user/ws_moveit2/install/moveit_servo/lib/libservo_node.so", at 0x7fc5e7bd5188, in moveit_servo::ServoNode::ServoNode(rclcpp::NodeOptions const&)
[servo_node-5] #8    Object "/home/user/ws_moveit2/install/moveit_servo/lib/libmoveit_servo_lib.so.2.5.5", at 0x7fc5e7ad028f, in moveit_servo::Servo::Servo(std::shared_ptr<rclcpp::Node> const&, std::shared_ptr<moveit_servo::ServoParameters const> const&, std::shared_ptr<planning_scene_monitor::PlanningSceneMonitor> const&)
[servo_node-5] #7    Object "/home/user/ws_moveit2/install/moveit_servo/lib/libmoveit_servo_lib.so.2.5.5", at 0x7fc5e7abc534, in moveit_servo::ServoCalcs::ServoCalcs(std::shared_ptr<rclcpp::Node> const&, std::shared_ptr<moveit_servo::ServoParameters const> const&, std::shared_ptr<planning_scene_monitor::PlanningSceneMonitor> const&) [clone .cold]
[servo_node-5] #6    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7fc5e763c4d7, in __cxa_throw
[servo_node-5] #5    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7fc5e763c276, in std::terminate()
[servo_node-5] #4    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7fc5e763c20b, in 
[servo_node-5] #3    Object "/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.30", at 0x7fc5e7630b9d, in 
[servo_node-5] #2    Source "./stdlib/abort.c", line 79, in abort [0x7fc5e736b7f2]
[servo_node-5] #1    Source "../sysdeps/posix/raise.c", line 26, in raise [0x7fc5e7385475]
[servo_node-5] #0  | Source "./nptl/pthread_kill.c", line 89, in __pthread_kill_internal
[servo_node-5]     | Source "./nptl/pthread_kill.c", line 78, in __pthread_kill_implementation
[servo_node-5]       Source "./nptl/pthread_kill.c", line 44, in __pthread_kill [0x7fc5e73d99fc]
[servo_node-5] Aborted (Signal sent by tkill() 9422 1000)
[move_group-4] [INFO] [1712761174.329595607] [moveit_ros.add_time_optimal_parameterization]: Param 'chomp.path_tolerance' was not set. Using default value: 0.100000
[move_group-4] [INFO] [1712761174.329634036] [moveit_ros.add_time_optimal_parameterization]: Param 'chomp.resample_dt' was not set. Using default value: 0.100000
[move_group-4] [INFO] [1712761174.329642029] [moveit_ros.add_time_optimal_parameterization]: Param 'chomp.min_angle_change' was not set. Using default value: 0.001000
[move_group-4] [INFO] [1712761174.329701770] [moveit_ros.fix_workspace_bounds]: Param 'chomp.default_workspace_bounds' was not set. Using default value: 10.000000
[move_group-4] [INFO] [1712761174.329724639] [moveit_ros.fix_start_state_bounds]: Param 'chomp.start_state_max_bounds_error' was set to 0.100000
[move_group-4] [INFO] [1712761174.329771518] [moveit_ros.fix_start_state_bounds]: Param 'chomp.start_state_max_dt' was not set. Using default value: 0.500000
[move_group-4] [INFO] [1712761174.329791547] [moveit_ros.fix_start_state_collision]: Param 'chomp.start_state_max_dt' was not set. Using default value: 0.500000
[move_group-4] [INFO] [1712761174.329800020] [moveit_ros.fix_start_state_collision]: Param 'chomp.jiggle_fraction' was set to 0.050000
[move_group-4] [INFO] [1712761174.329823832] [moveit_ros.fix_start_state_collision]: Param 'chomp.max_sampling_attempts' was not set. Using default value: 100
[move_group-4] [INFO] [1712761174.329840863] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Add Time Optimal Parameterization'
[move_group-4] [INFO] [1712761174.329849071] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Resolve constraint frames to robot links'
[move_group-4] [INFO] [1712761174.329853611] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Workspace Bounds'
[move_group-4] [INFO] [1712761174.329857971] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State Bounds'
[move_group-4] [INFO] [1712761174.329862102] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State In Collision'
[move_group-4] [INFO] [1712761174.329866181] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State Path Constraints'
[move_group-4] [INFO] [1712761174.365963201] [moveit.ros_planning_interface.moveit_cpp]: Loading planning pipeline 'ompl'
[ur_ros2_control_node-1] [INFO] [1712761174.398741075] [controller_manager]: Loading controller 'cartesian_force_controller'
[ERROR] [servo_node-5]: process has died [pid 9422, exit code -6, cmd '/home/user/ws_moveit2/install/moveit_servo/lib/moveit_servo/servo_node --ros-args -r __node:=servo_node --params-file /tmp/launch_params_mbtn2buf --params-file /tmp/launch_params_gt9ft_rt --params-file /tmp/launch_params_41e8kkgg --params-file /tmp/launch_params_98narlhm --params-file /tmp/launch_params_qg6o7kea'].
[move_group-4] [INFO] [1712761174.550640600] [moveit.ros_planning.planning_pipeline]: Using planning interface 'OMPL'
[move_group-4] [INFO] [1712761174.569025264] [moveit_ros.add_time_optimal_parameterization]: Param 'ompl.path_tolerance' was not set. Using default value: 0.100000
[move_group-4] [INFO] [1712761174.571077581] [moveit_ros.add_time_optimal_parameterization]: Param 'ompl.resample_dt' was not set. Using default value: 0.100000
[move_group-4] [INFO] [1712761174.571092876] [moveit_ros.add_time_optimal_parameterization]: Param 'ompl.min_angle_change' was not set. Using default value: 0.001000
[move_group-4] [INFO] [1712761174.571142036] [moveit_ros.fix_workspace_bounds]: Param 'ompl.default_workspace_bounds' was not set. Using default value: 10.000000
[move_group-4] [INFO] [1712761174.571165099] [moveit_ros.fix_start_state_bounds]: Param 'ompl.start_state_max_bounds_error' was set to 0.100000
[move_group-4] [INFO] [1712761174.571173486] [moveit_ros.fix_start_state_bounds]: Param 'ompl.start_state_max_dt' was not set. Using default value: 0.500000
[move_group-4] [INFO] [1712761174.571191127] [moveit_ros.fix_start_state_collision]: Param 'ompl.start_state_max_dt' was not set. Using default value: 0.500000
[move_group-4] [INFO] [1712761174.571199726] [moveit_ros.fix_start_state_collision]: Param 'ompl.jiggle_fraction' was set to 0.050000
[move_group-4] [INFO] [1712761174.571207150] [moveit_ros.fix_start_state_collision]: Param 'ompl.max_sampling_attempts' was not set. Using default value: 100
[move_group-4] [INFO] [1712761174.571223300] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Add Time Optimal Parameterization'
[move_group-4] [INFO] [1712761174.571229851] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Resolve constraint frames to robot links'
[move_group-4] [INFO] [1712761174.571234215] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Workspace Bounds'
[move_group-4] [INFO] [1712761174.571238712] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State Bounds'
[move_group-4] [INFO] [1712761174.571243117] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State In Collision'
[move_group-4] [INFO] [1712761174.571247412] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State Path Constraints'
[ur_ros2_control_node-1] [INFO] [1712761174.630039984] [controller_manager]: Loading controller 'scaled_joint_trajectory_controller'
[spawner-11] [INFO] [1712761174.665761631] [spawner_cartesian_force_controller]: Loaded cartesian_force_controller
[move_group-4] [INFO] [1712761174.943679218] [moveit.plugins.moveit_simple_controller_manager]: Added FollowJointTrajectory controller for scaled_joint_trajectory_controller
[move_group-4] [INFO] [1712761174.944315828] [moveit.plugins.moveit_simple_controller_manager]: Returned 1 controllers in list
[move_group-4] [INFO] [1712761174.944716605] [moveit.plugins.moveit_simple_controller_manager]: Returned 1 controllers in list
[move_group-4] [INFO] [1712761174.946822449] [moveit_ros.trajectory_execution_manager]: Trajectory execution is not managing controllers
[move_group-4] [INFO] [1712761174.946850132] [move_group.move_group]: MoveGroup debug mode is ON
[ur_ros2_control_node-1] [WARN] [1712761174.976839632] [scaled_joint_trajectory_controller]: [Deprecated]: "allow_nonzero_velocity_at_trajectory_end" is set to true. The default behavior will change to false.
[ur_ros2_control_node-1] [INFO] [1712761174.980488027] [controller_manager]: Loading controller 'forward_position_controller'
[spawner-3] [INFO] [1712761175.072138405] [spawner_scaled_joint_trajectory_controller]: Loaded scaled_joint_trajectory_controller
[ur_ros2_control_node-1] [INFO] [1712761175.140705565] [controller_manager]: Configuring controller 'cartesian_force_controller'
[move_group-4] [INFO] [1712761175.153409443] [move_group.move_group]: 
[move_group-4] 
[move_group-4] ********************************************************
[move_group-4] * MoveGroup using: 
[move_group-4] *     - ApplyPlanningSceneService
[move_group-4] *     - ClearOctomapService
[move_group-4] *     - ExecuteTaskSolution
[move_group-4] *     - CartesianPathService
[move_group-4] *     - ExecuteTrajectoryAction
[move_group-4] *     - GetPlanningSceneService
[move_group-4] *     - KinematicsService
[move_group-4] *     - MoveAction
[move_group-4] *     - MotionPlanService
[move_group-4] *     - QueryPlannersService
[move_group-4] *     - StateValidationService
[move_group-4] ********************************************************
[move_group-4] 
[move_group-4] [INFO] [1712761175.153655162] [moveit_move_group_capabilities_base.move_group_context]: MoveGroup context using planning plugin ompl_interface/OMPLPlanner
[move_group-4] [INFO] [1712761175.153668988] [moveit_move_group_capabilities_base.move_group_context]: MoveGroup context initialization complete
[move_group-4] Loading 'move_group/ApplyPlanningSceneService'...
[move_group-4] Loading 'move_group/ClearOctomapService'...
[move_group-4] Loading 'move_group/ExecuteTaskSolutionCapability'...
[move_group-4] Loading 'move_group/MoveGroupCartesianPathService'...
[move_group-4] Loading 'move_group/MoveGroupExecuteTrajectoryAction'...
[move_group-4] Loading 'move_group/MoveGroupGetPlanningSceneService'...
[move_group-4] Loading 'move_group/MoveGroupKinematicsService'...
[move_group-4] Loading 'move_group/MoveGroupMoveAction'...
[move_group-4] Loading 'move_group/MoveGroupPlanService'...
[move_group-4] Loading 'move_group/MoveGroupQueryPlannersService'...
[move_group-4] Loading 'move_group/MoveGroupStateValidationService'...
[move_group-4] 
[move_group-4] You can start planning now!
[move_group-4] 
[ur_ros2_control_node-1] [INFO] [1712761175.184948026] [cartesian_force_controller]: Forward dynamics solver initialized
[ur_ros2_control_node-1] [INFO] [1712761175.184991030] [cartesian_force_controller]: Forward dynamics solver has control over 6 joints
[spawner-10] [INFO] [1712761175.187991583] [spawner_forward_position_controller]: Loaded forward_position_controller
[ur_ros2_control_node-1] [INFO] [1712761175.290312081] [controller_manager]: Configuring controller 'forward_position_controller'
[ur_ros2_control_node-1] [INFO] [1712761175.295603861] [forward_position_controller]: configure successful
[ur_ros2_control_node-1] [INFO] [1712761175.298214932] [controller_manager]: Loading controller 'force_torque_sensor_broadcaster'
[ur_ros2_control_node-1] [INFO] [1712761175.344678614] [controller_manager]: Loading controller 'cartesian_compliance_controller'
[spawner-9] [INFO] [1712761175.387141287] [spawner_force_torque_sensor_broadcaster]: Loaded force_torque_sensor_broadcaster
[ur_ros2_control_node-1] [INFO] [1712761175.397826720] [controller_manager]: Configuring controller 'scaled_joint_trajectory_controller'
[ur_ros2_control_node-1] [INFO] [1712761175.398081323] [scaled_joint_trajectory_controller]: No specific joint names are used for command interfaces. Using 'joints' parameter.
[ur_ros2_control_node-1] [INFO] [1712761175.398110354] [scaled_joint_trajectory_controller]: Command interfaces are [position] and state interfaces are [position velocity].
[ur_ros2_control_node-1] [INFO] [1712761175.398127321] [scaled_joint_trajectory_controller]: Using 'splines' interpolation method.
[ur_ros2_control_node-1] [INFO] [1712761175.400243791] [scaled_joint_trajectory_controller]: Controller state will be published at 100.00 Hz.
[ur_ros2_control_node-1] [INFO] [1712761175.406828843] [scaled_joint_trajectory_controller]: Action status changes will be monitored at 20.00 Hz.
[ur_ros2_control_node-1] [INFO] [1712761175.419857271] [controller_manager]: Configuring controller 'force_torque_sensor_broadcaster'
[spawner-9] [INFO] [1712761175.431299575] [spawner_force_torque_sensor_broadcaster]: Configured and activated force_torque_sensor_broadcaster
[spawner-13] [INFO] [1712761175.433006402] [spawner_cartesian_compliance_controller]: Loaded cartesian_compliance_controller
[spawner-3] [INFO] [1712761175.435555512] [spawner_scaled_joint_trajectory_controller]: Configured and activated scaled_joint_trajectory_controller
[ur_ros2_control_node-1] [INFO] [1712761175.435825121] [controller_manager]: Configuring controller 'cartesian_compliance_controller'
[ur_ros2_control_node-1] [INFO] [1712761175.448158784] [cartesian_compliance_controller]: Forward dynamics solver initialized
[ur_ros2_control_node-1] [INFO] [1712761175.448204377] [cartesian_compliance_controller]: Forward dynamics solver has control over 6 joints
[INFO] [spawner-11]: process has finished cleanly [pid 9447]
[INFO] [spawner-10]: process has finished cleanly [pid 9445]
[INFO] [spawner-9]: process has finished cleanly [pid 9443]
[INFO] [spawner-3]: process has finished cleanly [pid 9417]
[INFO] [spawner-13]: process has finished cleanly [pid 9457]
[ur_ros2_control_node-1] [INFO] [1712761175.787847929] [controller_manager]: Loading controller 'joint_state_broadcaster'
[spawner-6] [INFO] [1712761175.840218088] [spawner_joint_state_broadcaster]: Loaded joint_state_broadcaster
[ur_ros2_control_node-1] [INFO] [1712761175.841927924] [controller_manager]: Configuring controller 'joint_state_broadcaster'
[ur_ros2_control_node-1] [INFO] [1712761175.842036578] [joint_state_broadcaster]: 'joints' or 'interfaces' parameter is empty. All available state interfaces will be published
[spawner-6] [INFO] [1712761175.853316318] [spawner_joint_state_broadcaster]: Configured and activated joint_state_broadcaster
[ur_ros2_control_node-1] [INFO] [1712761175.908473918] [controller_manager]: Loading controller 'speed_scaling_state_broadcaster'
[ur_ros2_control_node-1] [INFO] [1712761175.924715006] [speed_scaling_state_broadcaster]: Loading UR SpeedScalingStateBroadcaster with tf_prefix: 
[spawner-8] [INFO] [1712761175.957291839] [spawner_speed_scaling_state_broadcaster]: Loaded speed_scaling_state_broadcaster
[ur_ros2_control_node-1] [INFO] [1712761175.959633396] [controller_manager]: Configuring controller 'speed_scaling_state_broadcaster'
[ur_ros2_control_node-1] [INFO] [1712761175.959839956] [speed_scaling_state_broadcaster]: Publisher rate set to : 100.0 Hz
[spawner-8] [INFO] [1712761175.970618435] [spawner_speed_scaling_state_broadcaster]: Configured and activated speed_scaling_state_broadcaster
[ur_ros2_control_node-1] [INFO] [1712761175.974673820] [controller_manager]: Loading controller 'io_and_status_controller'
[INFO] [spawner-6]: process has finished cleanly [pid 9436]
[spawner-7] [INFO] [1712761176.037929984] [spawner_io_and_status_controller]: Loaded io_and_status_controller
[ur_ros2_control_node-1] [INFO] [1712761176.039656564] [controller_manager]: Configuring controller 'io_and_status_controller'
[spawner-7] [INFO] [1712761176.068528067] [spawner_io_and_status_controller]: Configured and activated io_and_status_controller
[INFO] [spawner-8]: process has finished cleanly [pid 9441]
[INFO] [spawner-7]: process has finished cleanly [pid 9439]
[ur_ros2_control_node-1] [INFO] [1712761176.348530463] [controller_manager]: Loading controller 'cartesian_motion_controller'
[spawner-12] [INFO] [1712761176.368802001] [spawner_cartesian_motion_controller]: Loaded cartesian_motion_controller
[ur_ros2_control_node-1] [INFO] [1712761176.451122613] [controller_manager]: Configuring controller 'cartesian_motion_controller'
[ur_ros2_control_node-1] [INFO] [1712761176.458533618] [cartesian_motion_controller]: Forward dynamics solver initialized
[ur_ros2_control_node-1] [INFO] [1712761176.458569048] [cartesian_motion_controller]: Forward dynamics solver has control over 6 joints
[INFO] [spawner-12]: process has finished cleanly [pid 9449]
[ur_ros2_control_node-1] [INFO] [1712761317.853781026] [forward_position_controller]: activate successful

`

**Comment by ravijo** [comment]
@shrutic12 

As said in above comments, old versions are using `servo_node_main`. And from ROS Iron, MoveIt2 made these changes. Please try running the Humble version. For more information, please feel free to read the comments above.
